test/unit/alignment/configuration/align_config_common_test.cpp

```diff
@@ -82,8 +82,3 @@ TYPED_TEST(alignment_configuration_test, configuration_exists_template)
     configuration cfg{TypeParam{}};
     helper_exists<TypeParam, decltype(cfg)>();
 }
-
```

Comment:
Why is this gone?

Reply:
Because the result config is no aggregate type anymore so the test fails... But it cannot be modelled as aggregate type so this requirement needs to be dropeed.

Reply:
I am just wondering why it didn't trigger an error for the `aligned_ends` config, which also has a user declared constructor. But alright. 
@eseiler Can this be resolved?